lib/pokemones.rb

```diff
@@ -1,6 +1,32 @@
 require "pokemones/version"
+require "httparty"
+require "json"
 
 module Pokemones
   class Error < StandardError; end
```

Comment:
Why 'Error' is the name of the class?

Reply:
It is automatically added when you create the gem with bundle. I was afraid to erase it, I thought it was needed for something haha